app/src/main/res/values/strings.xml

```diff
@@ -144,6 +144,7 @@
     <string name="contentDescriptionButtonDeleteDraft">Delete draft</string>
     <string name="contentDescriptionButtonDeleteHistory">Delete history entry</string>
     <string name="contentDescriptionButtonDeleteQuote">Delete quoted messages</string>
+    <string name="contentDescriptionButtonDeleteSearch">Clear search</string>
```

Comment:
Does the clear_text value in endIconMode already do something on its own for the content description or do we really need to manually set it? It looks like it actually already has its own content description